ministryofjustice/analytics-platform-ops
# Create a shell script that updates jupyter, rstudio and other user helm charts

## What

Added variable $HELM_CHART to specify any helm chart you like to update for a user
Added upgrade --install to upgrade HELM Charts rather than remove and then install

## How to review

1. Change script locally to point to your helm chart repo for jupyter, rstudio and a helm chart you want to update (e.g config-user)
2. run script on dev and see it update your helm charts with the latest Chart version
3. rollback to previous release to make sure rollback works

https://trello.com/c/P7BRQJWC/333-strip-out-ipython-notebook-output-using-a-git-hook-for-all-users
